routes/api/v1.js

```diff
@@ -223,9 +222,9 @@ api.declare({
     "Modify client `name`, `expires`, `scopes` and",
     "`description`.",
     "",
-    "**Required scopes**, in addition the scopes listed",
-    "above, the caller must also posses the all the scopes that is given to",
-    "the client that is updated."
+    "**Required scopes**: in addition the scopes listed",
+    "above, the `scopes` parameter must satisfy the caller's scopes.  The",
+    "client's existing scopes need not satisfy the caller's scopes."
```

Comment:
Again this is reversed...
It's the caller that must satisfy all the scopes assigned to the modified client. Also I'm not happy calling it the `scopes` parameter, it's the `scopes` property or `client.scopes` property, or `scopes` property of the payload. That's mostly nit...

However, direction of satisfiability is important.